unstructured/partition/docx.py

```diff
@@ -345,6 +351,15 @@ def page_number(self) -> int:
         """
         return self._page_counter
 
+    @lazyproperty
+    def strategy(self) -> str:
+        """The partitioning strategy for this document.
+
+        One of "hi_res", "fast", and a few others. These are available as class attributes on
+        `unstructured.partition.utils.constants.PartitionStrategy` but resolve to str values.
+        """
+        return PartitionStrategy.HI_RES if self._strategy is None else self._strategy
```

Comment:
Just want to double-check that it should default to hi_res, otherwise looks good

Reply:
k, yep, confirmed :)